src/hooks/useSearchQuery.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { getDailySummary, groupByDate, sortByDate } from '../utils';
+
+const url = process.env.REACT_APP_API_URL;
+const APPID = process.env.REACT_APP_API_KEY;
+
+
+const useSearchQuery = (query) => {
+  const [data, setData] = useState({ city: null, error: false, loading: false, list: [], summary: [] });
+
+  useEffect(() => {
+    const handleError = (error) => {
+      const message = typeof error === 'object' ? error.message : error;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
      const message = typeof error === "object" ? error.message : error;
```